[2022-12-16 10:54:02,437] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T00:31:00+00:00 [queued]>
[2022-12-16 10:54:02,775] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T00:31:00+00:00 [queued]>
[2022-12-16 10:54:02,775] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 10:54:02,785] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-16 10:54:02,785] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-16 10:54:03,168] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-12-16 00:31:00+00:00
[2022-12-16 10:54:03,308] {standard_task_runner.py:52} INFO - Started process 2752 to run task
[2022-12-16 10:54:03,302] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-12-16T00:31:00+00:00', '--job-id', '1417', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmp6602bzbw', '--error-file', '/tmp/tmprfme4cs8']
[2022-12-16 10:54:03,413] {standard_task_runner.py:80} INFO - Job 1417: Subtask get_last_id
[2022-12-16 10:54:04,881] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-12-16T00:31:00+00:00 [running]> on host 8e2f65ec4c74
[2022-12-16 10:54:07,187] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-12-16T00:31:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-16T00:31:00+00:00
[2022-12-16 10:54:07,613] {base.py:68} INFO - Using connection ID 'gp_conn' for task execution.
[2022-12-16 10:54:08,361] {python.py:173} INFO - Done. Returned value was: None
[2022-12-16 10:54:10,772] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221216T003100, start_date=20221216T105402, end_date=20221216T105410
[2022-12-16 10:54:11,325] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-16 10:54:12,521] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
